### PR TITLE
Update SNT FAQs

### PR DESCRIPTION
**Pull request by tferr**
- Fix Installation URL
- Update reference
- Minor

### CODE REVIEW

File: _pages/plugins/snt/faq.md [hunk 1]
```diff
@@ -9,19 +9,17 @@ nav-title: FAQ
...
 
-See [Installation details](#SNT#Installation).
+See [Installation details](/plugins/snt/index#installation).
 
 ### How do I cite SNT?
 
-SNT is based on multiple publications. To cite SNT:
-
-  - Ferreira T, et al. (**2019**) (TBD)
+  - Arshadi, C., Günther, U., Eddison, M. Harrington, K.I.S, Ferreira T. [SNT: a unifying toolbox for quantification of neuronal anatomy](https://doi.org/10.1038/s41592-021-01105-7). *Nature Methods* 18, 374–377 (2021).
```
Comment (ctrueden) [inline _pages/plugins/snt/faq.md:16]: You can use the `{% include citation doi=... %}` syntax for this. Or even better, put this new DOI in the front matter of the `plugins/snt/index` page, and then you can write `{% include citation id='plugins/snt' %}`.

See https://imagej.net/editing/citations

File: _pages/plugins/snt/faq.md [hunk 2]
```diff
@@ -9,19 +9,17 @@ nav-title: FAQ
...
+  - **[Sholl Analysis](/plugins/sholl-analysis)**: Ferreira T, Blackman A, Oyrer J, Jayabal A, Chung A, Watt A, Sjöström J, van Meyel D. [Neuronal morphometry directly from bitmap images](https://doi.org/10.1038/nmeth.3125), *Nature Methods* 11(10): 982–984 (2014).
   - **[Tubular Geodesics](/plugins/snt/tubular-geodesics)**: Türetken E, Benmansour F, Fua P. [Automated Reconstruction of Tree Structures using Path Classifiers and Mixed Integer Programming](https://infoscience.epfl.ch/record/176222/files/turetken_et_al_2012.pdf?version=1). <i>IEEE Conference on Computer Vision and Pattern Recognition</i>, Providence, Rhode Island, 2012.
-  - **[Cx3D](/plugins/snt/modeling)**: Zubler, F. & Douglas, R. [A framework for modeling the growth and development of neurons and networks](https://doi.org/10.3389/neuro.10.025.2009). Front. Comput. Neurosci. 3, 25 (2009)\].
+  - **[Cx3D](/plugins/snt/modeling)**: Zubler, F. & Douglas, R. [A framework for modeling the growth and development of neurons and networks](https://doi.org/10.3389/neuro.10.025.2009). Front. Comput. Neurosci. 3, 25 (2009).
```
Comment (ctrueden) [inline _pages/plugins/snt/faq.md:22]: Again, since these all have DOIs, you can write e.g. `{% include citation doi='10.3389/neuro.10.025.2009' %}`.

**Review by ctrueden — COMMENTED** [review]
Just a couple of quick comments about the wiki's new citation capabilities.

File: _pages/plugins/snt/faq.md (hunk 1 above)
Comment (hinerm) [reply _pages/plugins/snt/faq.md:16]: @tferr this sounds useful - is it something you want to do as part of this pr?

**Pull request merged**